usl_pipeline/cloud_functions/requirements.txt

```diff
@@ -2,6 +2,7 @@ affine==2.4.0
 attrs==23.2.0
 blinker==1.7.0
 cachetools==5.3.3
+cartopy==0.23.0
```

Comment:
are there no addition dependencies-of-dependencies to include?

Reply:
shoot missed this - for wrf python in particular?

I could do a pip freeze as a follow up. we just want to exclude the packages we're using for development purposes rihgt?

Reply:
right, I usually do
```bash
rm venv
virtualenv venv
source venv/bin/activate
pip install -r requirements.txt <my-new-package>
pip freeze > requirements.txt
```